toolkit/tools/imagecustomizerschemacli/schema.json

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
Should this file live in the `imagecustomizerapi` directory?

Reply:
I went with updating the makefile to keep the command simple.